extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetColumnLocation.java

```diff
@@ -162,15 +162,24 @@ public <METADATA_TYPE> METADATA_TYPE getMetadata(
                     : groupingColumnInfo.groupingTablePath();
             final String groupingFilePath =
                     parquetFile.toPath().getParent().resolve(groupingFileRelativePath).toString();
-            final ParquetFileReader parquetFileReader;
+            ParquetFileReader parquetFileReader;
             try {
                 parquetFileReader = new ParquetFileReader(groupingFilePath, tl().getChannelProvider());
-            } catch (IOException e) {
-                log.warn().append("Failed to read expected grouping file ").append(groupingFilePath)
-                        .append(" for table location ").append(tl()).append(", column ")
-                        .append(getName())
-                        .endl();
-                return null;
+            } catch (final UncheckedIOException e1) {
```

Comment:
This isn't quite right. We would only want to fall back to legacy default if we had no explicit grouping column info, that is if `groupingColumnInfo == null`.
So the logic should be:
```
if groupingColumnInfo != null {
    try to load from there
    // no fallback
} else {
    try to load from new default path
    if that fails {
        try to load from legacy default path
    }
}
```